src/test/splitting/tasks_tests_handler/python_parsing_test.py

```diff
@@ -2,11 +2,11 @@
 
 import os
 import sys
+import pytest
```

Comment:
Pytest is not a standard library, so please, move this import into a separated second group